src/basket/styles/basket-mini-view.module.scss

```diff
@@ -0,0 +1,28 @@
+@import 'franklin-sites/src/styles/settings';
```

Comment:
Maybe for another time, but I wonder if there should be modifiers defined in franklin for this. We already have the "compact" table.

Reply:
For the settings? I'm not sure what you are referring to

Reply:
> I mean most of the file - it seems that it's related to making the table even more compact, removing margins etc... so thought it could be something useful in other places too and could be part of the component definition.

Following the thread here otherwise it's hard to follow.

The issue is that it's overriding styles in sliding-panel, cards, tabs (tabs headers specifically), data-table, and button-groups. I agree the content of this file shouldn't have to exist, but I don't know where else we'd put it as it covers so many different franklin components interacting with each other, and just for this specific basket use case.